packages/git/src/browser/git-view-contribution.ts

```diff
@@ -261,6 +263,7 @@ export class GitViewContribution extends AbstractViewContribution<GitWidget>
             isVisible: () => this.syncService.canPublish()
         });
         registry.registerCommand(GIT_COMMANDS.CLONE, {
+            isEnabled: () => !!this.workspaceService.tryGetRoots().length,
```

Comment:
would `this.workspaceService.opened` work?

Reply:
yes that's shorter :)